can/bus.py

```diff
@@ -56,6 +66,11 @@ def __init__(self, channel, can_filters=None, **kwargs):
         self._periodic_tasks = []
         self.set_filters(can_filters)
 
+        #: initialize Error status, transmit error count TEC, receive error count REC
+        self.error_state: ErrorState = ErrorState.ERROR_ACTIVE
+        self.TEC: int = -1
```

Comment:
I would spell them out and document these parameters since they are "public".

Reply:
You're right. I changed it to txErrorCount and rxErrorCount.